vagga.yaml

```diff
@@ -24,3 +46,26 @@ commands:
     description: Shortcut to manage.py
     container: django
     run: [python3, manage.py]
+  run-postgres: !Supervise
+    description: Start the django development server using Postgres database
+    children:
+      app: !Command
+        container: django
+        environ:
+          DATABASE_URL: postgresql://vagga:vagga@127.0.0.1:5433/django
+        run: |
+            touch /work/.dbcreation # Create lock file
+            while [ -f /work/.dbcreation ]; do sleep 0.2; done # Acquire lock
+            python3 manage.py migrate
+            python3 manage.py runserver --insecure
+      db: !Command
+        container: postgres
+        run: |
+            chown postgres:postgres $PGDATA;
+            su postgres -c "$PG_BIN/pg_ctl initdb";
```

Comment:
I think, DB creation should not repeat on every command run, you need a separate command for initializing the db